package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "floeproject.org",
+  "version": "1.0.0",
+  "description": "FLOE Project Website",
+  "scripts": {
+    "build": "eleventy",
+    "eleventy": "eleventy --serve",
+    "eleventyport": "eleventy --serve --port=9778"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fluid-project/floeproject.org.git"
+  },
+  "keywords": [
+    "Floe",
+    "Project",
+    "Flexible",
+    "Learning",
+    "for",
+    "Open",
+    "Education"
+  ],
+  "author": "OCAD University <idrc@ocadu.ca>",
+  "license": "CREATIVE COMMONS ATTRIBUTION 4.0 INTERNATIONAL LICENSE",
```

Comment:
The SPDX license string for this license is "CC-BY-4.0".

Reply:
Fixed